core/web/resolver/spec.go

```diff
@@ -332,6 +332,11 @@ func (r *OCRSpecResolver) EVMChainID() *string {
 	return &chainID
 }
 
+// ContractConfigTrackerPollInterval resolves the spec's database timeout.
+func (r *OCRSpecResolver) OCRDatabaseTimeout() string {
```

Comment:
For this to resolve you will also need to add the value to the `graphql` spec.

You can find this in `core/web/schema/type/spec.graphql`. 

```
ocrDatabaseTimeout: String!
```

Reply:
Fixed.